tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -1885,7 +1892,7 @@ public function listFolder(
 			$this->getStepLineRef(),
 			$properties,
 			$type,
-			($this->usingOldDavPath) ? 1 : 2
+			$davPathVersion
```

Comment:
```suggestion
			$this->getDavPathVersion()
```
IMO it's good to just provide the DAV path here rather introducing the function param.